subsys/settings/Kconfig

```diff
@@ -63,6 +63,24 @@ config SETTINGS_NVS
 	help
 	  Enables NVS storage support
 
+if SETTINGS_NVS
+
+config SETTINGS_NVS_CACHE
+	bool "Non-volatile Storage cache"
```

Comment:
That's not a proper prompt. Should we rename the Kconfig to sth like `SETTINGS_NVS_NAME_CACHE` and change the prompt to "NVS name lookup cache" to better describe the purpose?